x-pack/plugins/alerting/server/routes/get_global_execution_kpi.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { IRouter } from '@kbn/core/server';
+import { schema } from '@kbn/config-schema';
+import { AlertingRequestHandlerContext, INTERNAL_BASE_ALERTING_API_PATH } from '../types';
+import { RewriteRequestCase, verifyAccessAndContext } from './lib';
+import { GetGlobalExecutionKPIParams } from '../rules_client';
+import { ILicenseState } from '../lib';
+
+const querySchema = schema.object({
+  date_start: schema.string(),
+  date_end: schema.maybe(schema.string()),
+  filter: schema.maybe(schema.string()),
+});
+
+const rewriteReq: RewriteRequestCase<GetGlobalExecutionKPIParams> = ({
+  date_start: dateStart,
+  date_end: dateEnd,
+  ...rest
+}) => ({
+  ...rest,
+  dateStart,
+  dateEnd,
+});
+
+export const getGlobalExecutionKPIRoute = (
+  router: IRouter<AlertingRequestHandlerContext>,
+  licenseState: ILicenseState
+) => {
+  router.get(
+    {
+      path: `${INTERNAL_BASE_ALERTING_API_PATH}/rule/{id}/_global_execution_kpi`,
```

Comment:
Do you need an ID for the global execution KPI route?

Reply:
I fixed that!